src/discussion/management/commands/backfill_paper_to_grm.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+
+from django.core.management.base import BaseCommand
+
+from discussion.reaction_models import Vote as GrmVote
+from paper.models import Paper
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        today = datetime.datetime.now()
+        paper_sync_stop_date = datetime.datetime(
+            year=2019, month=1, day=1, hour=0, minute=0, second=0
+        )
+        papers = Paper.objects.filter(
+            created_date__lte=today,
+            created_date__gte=paper_sync_stop_date,
+        ).order_by("created_date")
+        count = papers.count()
+        for i, paper in enumerate(papers.iterator()):
+            print(f"{i + 1}/{count} - paper_id {paper.id}")
+            legacy_votes = paper.votes_legacy
+            for j, vote_legacy in enumerate(legacy_votes.iterator()):
+                try:
+                    grm_vote = GrmVote(
+                        created_by=vote_legacy.created_by,
+                        created_date=vote_legacy.created_date,
+                        item=paper,
+                        updated_date=vote_legacy.created_date,
+                        vote_type=vote_legacy.vote_type,
+                    )
+                    grm_vote.save()
```

Comment:
I think this might be slightly better?
```suggestion
from paper.models import Vote as PaperVote

for i, paper_vote in enumerate(PaperVote.objects.all()):
    ...
    GrmVote.objects.create(
        ...
        item=paper_vote.paper
    )
```